### PR TITLE
update sample config for kafka 0.8.2

### PR DESCRIPTION
**Pull request by aybabtme**
@brian-brazil: as we discussed in #prometheus, I've updated the example rules for kafka 0.8.2

### CODE REVIEW

File: example_configs/kafka.json [hunk 1]
```diff
@@ -1,85 +1,107 @@
...
+       "partition": "$5"
+     }
+   },
+   {"pattern": "kafka.(\w+)<type=(.+), name=(.+)><>(Count|Value)",
+     "name": "kafka_$1_$2_$3_$4"
+   },
+   {"pattern": "kafka.(\w+)<type=(.+), name=(.+), (\w+)=(.+)><>(Count|Value)",
+     "name": "kafka_$1_$2_$3_$6",
+     "labels": {
+       "$4": "$5"
```
Comment (aybabtme) [inline example_configs/kafka.json]: I'm not sure if this is possible =/

Comment (brian-brazil) [inline example_configs/kafka.json]: That's possible, and handy to collapse things down

**Comment by brian-brazil** [comment]
We'll want to keep both version of the config around, can you suffix one with pre0.8.2 and the other with 0.8.2 ?

**Comment by brian-brazil** [comment]
Has this been run on a kafkas that are leaders and non-leaders? If you could gist the full /metrics output that'd be handy

**Comment by brian-brazil** [comment]
Has this been run on a kafkas that are leaders and non-leaders? If you could gist the full /metrics output that'd be handy to check it's all looking good.

**Comment by aybabtme** [comment]
I'm deploying it to a staging environment, I'll loop back with more details.

**Comment by robbiet480** [comment]
This is working great for me on leaders and non-leaders in production

**Comment by brian-brazil** [comment]
Okay, can the files be renamed then so we can keep the pre0.8.2 config around for anyone who needs it?

**Comment by aybabtme** [comment]
@brian-brazil sorry I completely lost track of this, I'll rename the file.

**Comment by aybabtme** [comment]
renamed the files to be:

```
kafka-0-8-2.json
kafka-pre0-8-2.json
```

**Comment by brian-brazil** [comment]
Thanks!

**Comment by robbiet480** [comment]
Thanks all!

**Pull request merged**